tests/lib/query_model_utils.py

```diff
@@ -2,13 +2,16 @@
 
 from databuilder.query_model import nodes as query_model
 
+# These are only exercised during the long-running generative tests when
+# GENTEST_COMPREHENSIVE is enabled
```

Comment:
Is it worth adding a custom coverage pragma for things that are only hit in generative tests?  Or E_LIFETOOSHORT?

Reply:
I don't think it's worth it when we're just talking about coverage of test code. There are all sorts of reasons that not every bit of test code gets exercised on every test run and so we're generally more liberal with no-cover here.